delineate.py

```diff
@@ -324,26 +328,27 @@ def plot_basins(suffix: str):
     # No longer needed with GeoPandas v 0.14
     #gages_df.drop(['geometry'], axis=1, inplace=True)
 
-    if VERBOSE: print("Finding out which Level 2 megabasin(s) your points are in")
+    if conf.VERBOSE: 
+        print("Finding out which Level 2 megabasin(s) your points are in")
     # This file has the merged "megabasins_gdf" in it
-    merit_basins_shp = 'data/shp/basins_level2/merit_hydro_vect_level2.shp'
-    megabasins_gdf = gpd.read_file(merit_basins_shp)
+    #merit_basins_shp = 'data/shp/basins_level2/merit_hydro_vect_level2.shp'
+    megabasins_gdf = gpd.read_file(conf.MERIT_BASINS_SHP)
```

Comment:
This is the change I added to get rid of the hardcoded path in the middle of the path. Links to my last comment